allenact/base_abstractions/task.py

```diff
@@ -231,7 +231,7 @@ def metrics(self) -> Dict[str, Any]:
         }
 
     def query_expert(self, **kwargs) -> Tuple[Any, bool]:
-        """Query the expert policy for this task.
+        """(Deprecated) Query the expert policy for this task.
```

Comment:
Given this is deprecated, can you point people to the `query_expert` method of the expert action sensor and give a  brief description of the right way to do things?

Reply:
I will 👍